### PR TITLE
[docstring] fix docstring DPRConfig

### PR DESCRIPTION
**Pull request by AVAniketh0905**
# What does this PR do?

Fixes #26638 only for `DPRConfig`

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@ydshieh 

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/dpr/configuration_dpr.py [hunk 1]
```diff
@@ -83,6 +83,8 @@ class DPRConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        pad_token_id (`int`, *optional*, defaults to 0):
+            The pad token id of the model.
```
Comment (ydshieh) [inline src/transformers/models/dpr/configuration_dpr.py:87]:
```suggestion
            Padding token id.
```

**Review by ydshieh — COMMENTED** [review]
Thank you @AVAniketh0905 ❤️ .

A tiny suggestion and ready to go!

**Review by ydshieh — APPROVED** [review]
Thank you!

**Comment by AVAniketh0905** [comment]
> Thank you!

Thank you very much!!!

**Pull request merged**